internal/constants/constants.go

```diff
@@ -23,3 +23,15 @@ const (
 const (
 	MaxDepth = 4
 )
+const (
+	UserRoleName       = "UserRole"
+	SuperAdminRoleName = "SuperAdminRole"
+)
+const (
+	ErrorFromRedisCache   = "RedisCache Error"
+	ErrorFromGetRole      = "RedisCache GetRole Error"
+	ErrorUnauthorizedUser = "Unauthorized User"
+	ErrorFromCreateRole   = "CreateRole Error"
+	SuccessCase           = "Success"
+	ErrorFindingUser      = "Fail on finding user"
+)
```

Comment:
Lets not have test related constants in this file. Move respective constants to individual test files